fhir_extractor/src/analyze/concept_map.py

```diff
@@ -1,11 +1,14 @@
 from typing import TypeVar
-import requests
-import numpy as np
 import logging
+import numpy as np
+import os
+import requests
 
 import fhir_extractor.src
 from fhir_extractor.src.errors import OperationOutcome
 
+API_URL = os.getenv("API_URL")
```

Comment:
maybe PYROG_API_URL (since we can have different API, the extractor having an API as well)

Reply:
You're right